dds/DCPS/RTPS/Spdp.cpp

```diff
@@ -1705,15 +1705,15 @@ Spdp::SpdpTransport::handle_input(ACE_HANDLE h)
   if (bytes > 0) {
     buff_.wr_ptr(bytes);
   } else if (bytes == 0) {
-    return -1;
+    return 0;
   } else {
     ACE_DEBUG((
-          LM_ERROR,
-          ACE_TEXT("(%P|%t) ERROR: Spdp::SpdpTransport::handle_input() - ")
+          LM_WARNING,
+          ACE_TEXT("(%P|%t) WARNING: Spdp::SpdpTransport::handle_input() - ")
           ACE_TEXT("error reading from %C socket %p\n")
```

Comment:
Was this impacting tests, or some other issue? Do we need to remove the word error from the debug message as well?

Reply:
It impacts the new tests for RtpsRelay.  We observed "connection reset" errors.  Keeping error in the text is appropriate.

Reply:
https://github.com/DOCGroup/autobuild/blob/891e9c5fe2bab56ca54a4ecd8f267f574aca55c0/common/prettify.pm#L1468

Reply:
Gotcha. Always wondered about the full list. Now I know not to make any debug messages about 1970/80's cartoon show "fatalbert" or the value of my eclectically-named constant "INISFAIL:"